src/dftbp/common/file.F90

```diff
@@ -251,6 +255,12 @@ subroutine TFileDescr_connectToFile(this, file, options, mode, ioStat, ioMsg)
       end if
     end if
 
+    if (present(parallelWriting)) then
```

Comment:
Would

parallelWriting_ = .false.
if (present(parallelWriting)) parallelWriting_ = parallelWriting_

Also be OK, but more concise?